packages/legend-transac/src/@types/event/events.ts

```diff
@@ -275,6 +275,12 @@ export interface EventPayload {
         userId: string;
         userToUnblockId: string;
     };
+    /**
+     * Event to update the social user
+     */
+    'social.update_user': {
```

Comment:
The name `update_user` implies this is a command directing other microservices to take action. Since this is actually an event notifying that a user was already updated, we renamed it to `updated_user` to better reflect its purpose.